src/less/field/base/field.less

```diff
@@ -53,7 +53,7 @@ div.field__group {
 }
 
 .field__description {
-    font-size: @ds4-font-size-12;
+    font-size: @font-size-12;
```

Comment:
Why do we still have font-size-12?   I thought we moved all these to DS-specific labels?

Reply:
Let me check that. I'll fix it.